sdk/auth.js

```diff
@@ -20,6 +20,7 @@ const verifyAPIKey = async function verifyAPIKey(apiKey) {
       headers: {
         'X-Binaris-Api-Key': apiKey,
       },
+      family: 4,
```

Comment:
I think no reason to use raw request here,  possible to use `getValidatedBody` to get all the benefits from reuse?